src/plugins/intel_gpu/src/graph/concatenation.cpp

```diff
@@ -116,6 +116,10 @@ concatenation_inst::typed_primitive_inst(network& network, concatenation_node co
     for (const auto& i : node.get_dependencies()) {
         auto input_i_layout = i.first->get_output_layout();
         auto input_mem_size = input_i_layout.get_dims();
+        if (i.first->get_outputs_count() > 1 && i.second > 0) {
+            input_i_layout = i.first->get_output_layout(false, i.second);
+            input_mem_size = input_i_layout.get_dims();
+        }
```

Comment:
I think that can be changed to
```cpp
        auto input_i_layout = i.first->get_output_layout(false, i.second);
        auto input_mem_size = input_i_layout.get_dims();
```
isn't it?

Reply:
done